### PR TITLE
Issue #13213: Removed //ok from illegalinstantiation

### PR DESCRIPTION
**Pull request by akankshat05**
For issue #13213 
Remove ok comments from below files :
- InputIllegalInstantiation.java
- InputIllegalInstantiationBeginTree1.java
- InputIllegalInstantiationBeginTree2.java
- InputIllegalInstantiationLang3.java
- InputIllegalInstantiationNameSimilarToStandardClasses.java
- InputIllegalInstantiationSameClassNameJavaLang.java
- InputIllegalInstantiationSemantic.java

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalinstantiation/InputIllegalInstantiationBeginTree1.java
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalinstantiation/InputIllegalInstantiationBeginTree1.java:3]: Everything lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
There is a failing CI.. I am however unable to see the error log.

> Please restart the checks.. They are triggered by a push

Actually, lets wait it out, the failure seems to affect other PRs too.

**Comment by romani** [comment]
please resolve conflict and rebase on our latest master code.

**Comment by akankshat05** [comment]
@romani please review

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalinstantiation/InputIllegalInstantiationBeginTree2.java
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalinstantiation/InputIllegalInstantiationBeginTree2.java:10]: Conflict is resolved as all CI are green

**Review by MANISH-K-07 — COMMENTED** [review]
Please wait for approval....

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**
